openmdao/docs/openmdao_book/advanced_user_guide/jax_derivatives/partial_derivs_explicit.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>nitpick: can you link to JAX the first time it's mentioned so users can check it out easily if they don't know what it is?</p><p> </p><p>Can you clarify what "tall" and "wide" matrices mean practically? Maybe change it to: "Forward-mode is better for "tall" Jacobian matrices (more outputs than inputs) whereas reverse-mode is better for "wide" Jacobian matrices (more inputs than outputs)."</p><p>That could be wordy and you might want to break up the sentence, but I want to explain to less mathy users what that means.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2933/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>